src/layout/MainLayout.tsx

```
@@ -1,67 +1,13 @@
-import { useState } from 'react';
-import { Link, Outlet, useLocation } from 'react-router-dom';
-
-import SnackIcon from '@/assets/snack.svg?react';
-import LoginModal from '@/shared/components/loginModal/LoginModal';
+import { Outlet } from 'react-router-dom';
+import Navbar from '@/shared/components/Navbar/Navbar';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)